stable/prometheus/templates/alertmanager-deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
 {{- if .Values.alertmanager.priorityClassName }}
       priorityClassName: "{{ .Values.alertmanager.priorityClassName }}"
 {{- end }}
+      securityContext:
+        runAsUser: 65534
+        fsGroup: 65534
```

Comment:
You should probably set `runAsGroup: 65534` as well since it's enabled by default in 1.14 and it doesn't hurt setting it on clusters where it's not enabled.